assert/assertions_test.go

```diff
@@ -1576,30 +1576,43 @@ func TestPanicsWithValue(t *testing.T) {
 func TestPanicsWithError(t *testing.T) {
 	t.Parallel()
 
-	mockT := new(testing.T)
-
-	if !PanicsWithError(mockT, "panic", func() {
+	mckT := new(captureTestingT)
```

Comment:
Rename mckT to mockT for consistency with other tests.

Reply:
Good catch, amended!